### PR TITLE
Auth fix, and small List fix

### PR DESCRIPTION
**Pull request by carlos-jmh**
### Notify

r? @evoth @diti85 
cc 

### Summary

- Auth fixeed (sign out, household not updating, etc...)
- Small List Item deletion fix

### Motivation

MVP.

### Testing

Manual.

### Rollout/monitoring/revert plan

Can rollback to previous commit.

### CODE REVIEW

**Review by evoth — DISMISSED** [review]
LGTM!

File: src/components/home/lists/List.js
Comment (evoth) [inline src/components/home/lists/List.js:1]: While this does make the change occur immediately in the UI, I found while testing that after checking/unchecking a single item, updating items will break entirely for that list (idk why). My previous approach was to just let the subscription update it, but maybe you can fix the bug?

File: src/components/home/lists/List.js [hunk 1]
```diff
@@ -44,11 +44,13 @@ export default function List({ list }) {
...
-    //     oldItem.id == item.id ? updatedItem : oldItem
-    //   );
-    // });
+    updatedItem._version += 1;
+
+    setItems((oldItems) => {
+      return oldItems.map((oldItem) =>
+        oldItem.id == item.id ? updatedItem : oldItem
+      );
+    });
```
Comment (evoth) [inline src/components/home/lists/List.js:53]: While this does make the change occur immediately in the UI, I found while testing that after checking/unchecking a single item, updating items will break entirely for that list (idk why). My previous approach was to just let the subscription update it, but maybe you can fix the bug?

Comment (carlos-jmh) [reply src/components/home/lists/List.js:53]: Will check this out.
Without this addition, checking itmes was not working dor me

**Review by evoth — APPROVED** [review]
LGTM!

**Pull request merged**